src/main/docs/guide/security/authenticationStrategies/jwt/reader/bearerToken.adoc

```diff
@@ -1,25 +1,21 @@
-Micronaut supports https://tools.ietf.org/html/rfc6750[RFC 6750 Bearer Token] specification out-of-the-box.
-
-It is enabled by default but requires JWT Authentication to be enabled.
-
-The following sequence illustrates the authentication flow:
+Micronaut supports the https://tools.ietf.org/html/rfc6750[RFC 6750 Bearer Token] specification for transmitting JWT tokens. The following sequence illustrates the RFC 6750 authentication flow:
 
 image::jwt-bearer-token.svg[]
 
-The following configuration properties are available to customize Bearer Token read:
+The following configuration properties are available to customize how the Bearer Token will be embedded in the request:
 
 include::{includedir}configurationProperties/io.micronaut.security.token.jwt.bearer.BearerTokenConfigurationProperties.adoc[]
 
 == Sending tokens in the request
 
-The following code snippet illustrates how to send a JWT token in the `Authorization` request header:
+The code snippet below illustrates how to send a JWT token in the `Authorization` request header:
 
-[source, bash]
-.Accessing a protected resource using Authorization request header
+[source, groovy]
+.Accessing a protected resource using an Authorization request header
 ----
-GET /protectedResource HTTP/1.1
-Host: micronaut.example
-Authorization: Bearer eyJhbGciOiJIUzI1NiJ9.eyJleHAiOjE0MjI5OTU5MjIsInN1YiI6ImppbWkiLCJyb2xlcyI6WyJST0xFX0FETUlOIiwiUk9MRV9VU0VSIl0sImlhdCI6MTQyMjk5MjMyMn0.rA7A2Gwt14LaYMpxNRtrCdO24RGrfHtZXY9fIjV8x8o
+String accessToken = //... obtain the access token
```

Comment:
I will not remove the code which shows the request without micronaut code. If you want to add a code example. I will use the method: `HttpRequest.GET('/' ).bearer(accessToken)`

Reply:
My issue with the existing code snippet was that (in my opinion) it didn't do what the docs claimed: "[show] how to send a JWT token in the Authorization request header". The snippet is just a raw HTTP request with the auth header. It's not something one would ever write in a Micronaut app. 

If we're wanting to show what a request auth header looks like, that's fine with me, but I still think we should include a code example that uses Micronaut's client to make the request with the header and token.